### PR TITLE
bureau flag added and Bureau menu wrapped in it

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
💭 not sure if this is how we add a flag 🤔

### CODE REVIEW

File: public/config/config.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "bid_stats": false,
     "client_profiles": true,
     "tandem": true,
+    "bureau": true,
```
Comment (mjoyce91) [inline public/config/config.json:22]: add to `config_dev.json` as well so that it appears in AWS

**Review by mjoyce91 — COMMENTED** [review]
> 💭 not sure if this is how we add a flag 🤔

Looks good! Just see my one comment

File: public/config/config.json (hunk 1 above)
Comment (elizabeth-jimenez) [reply public/config/config.json:22]: ahhh, gotcha! Done

**Review by scott062 — APPROVED** [review]
🎏

**Pull request merged**